prisma/schema.sql

```diff
@@ -0,0 +1,193 @@
+);
+
+-- Create physicians table
+CREATE TABLE physicians (
+  id            SERIAL PRIMARY KEY NOT NULL,
+  first_name    VARCHAR(255) NOT NULL,
+  last_name     VARCHAR(255) NOT NULL,
+  msp_number    INTEGER UNIQUE NOT NULL,
+  address       VARCHAR(255) NOT NULL,
+  city          VARCHAR(255) NOT NULL,
+  province      Province NOT NULL,
+  postal_code   CHAR(6) NOT NULL,
+  phone         VARCHAR(50) NOT NULL,
+  status        PhysicianStatus NOT NULL,
+  notes         TEXT,
+  created_at    TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at    TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create applicants table
+CREATE TABLE applicants (
+  id                        SERIAL PRIMARY KEY NOT NULL,
+  first_name                VARCHAR(255) NOT NULL,
+  middle_name               VARCHAR(255),
+  last_name                 VARCHAR(255) NOT NULL,
+  date_of_birth             DATE NOT NULL,
+  gender                    Gender NOT NULL,
+  email                     VARCHAR(255) UNIQUE,
+  phone                     VARCHAR(50) NOT NULL,
+  province                  Province NOT NULL,
+  city                      VARCHAR(255) NOT NULL,
+  address                   VARCHAR(255) NOT NULL,
+  postal_code               CHAR(6) NOT NULL,
+  rcd_user_id               INTEGER UNIQUE,
+  status                    ApplicantStatus,
+  created_at                TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at                TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create guardians table
+CREATE TABLE guardians (
+  id              SERIAL PRIMARY KEY NOT NULL,
+  first_name      VARCHAR(255) NOT NULL, 
+  middle_name     VARCHAR(255),
+  last_name       VARCHAR(255) NOT NULL,
+  address         VARCHAR(255) NOT NULL,
+  city            VARCHAR(255) NOT NULL,
+  province        Province NOT NULL,
+  postal_code     CHAR(6) NOT NULL,
+  phone           VARCHAR(50) NOT NULL,
+  relationship    VARCHAR(255) NOT NULL,
+  notes           TEXT,
+  applicant_id    INTEGER UNIQUE NOT NULL,
+  created_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  FOREIGN KEY(applicant_id) REFERENCES applicants(id)
+);
+
+-- Create medical information table
+CREATE TABLE medical_information (
+  id                       SERIAL PRIMARY KEY NOT NULL,
+  disability               VARCHAR(255) NOT NULL,
+  affects_mobility         BOOLEAN NOT NULL DEFAULT false,
+  mobility_aid_required    BOOLEAN NOT NULL DEFAULT false,
+  cannot_walk_100m         BOOLEAN NOT NULL DEFAULT false,
+  notes                    TEXT,
+  certification_date       DATE,
+  aid                      Aid ARRAY,
+  physician_id             INTEGER NOT NULL,
+  applicant_id             INTEGER UNIQUE NOT NULL,
+  created_at               TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at               TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
```

Comment:
Wait did these newlines not affect the execution of the script?

Reply:
I just tried out the SQL script to create a new db from scratch and it worked so these line breaks aren't an issue